src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -131,7 +133,9 @@ private void StartWorkerProcesses(int startIndex, Action startAction)
         {
             for (var count = startIndex; count < _maxProcessCount; count++)
             {
-                startAction = startAction.Debounce(count * _debounceSeconds * 1000);
+                CancellationTokenSource startTokenSource = new CancellationTokenSource();
```

Comment:
If start and cancellation is always done for all channels, is there a reason why we need to keep a `CancellationTokenSource` for each one of them? Couldn't we reuse the same and signal a single token?

Reply:
You are right! Fixed it.